docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -9,6 +9,25 @@ info:
     See [the API overview](../api/) for an introduction to the API and information
     on how authorization works.
 
+    ## Expanding Resources
+    Resources often contain links to related resources in their response bodies.
+    Some endpoints support resource expansion, allowing those related resources
+    to be expanded inline in the response.
+
+    To expand resources (where supported), use the `expand` request parameter.
+    Individual endpoint documentation indicates which resources are expandable.
+
+    The `expand` parameter value is an Array:
+
+    ```
+    GET /api/things?expand=organizations&expand=foobars
+    ```
+
+    would expand `organization` and `foobar` relationships within the
+    (hypothetical) returned `thing` objects.
+
+    `expand` values should be the plural of the noun being expanded.
```

Comment:
I still think I prefer the approach taken in [Stripe's API](https://stripe.com/docs/api#expanding_objects) where the `expand` value is just the path to the field, for two reasons. First, this provides a natural way to handle nesting (as their docs explain. We don't need it right now, but it seems like a logical thing to want). Secondly they don't need to spell out what the pluralization is, they just mark individual fields as "Expandable" if supported. Thoughts?

Reply:
Good feedback! I think I'm inclined to agree now that I look more closely. To clarify, however:

> the expand value is just the path to the field

Being ever-so-slightly pedantic, I want to be sure you mean:

> a valid expand value is a path to a field (in the pertinent object)

That is, `expand` is still OK to be an Array, correct?

Reply:
Yes, indeed.

Reply:
@robertknight Ready for re-review!